src/EditorFeatures/VisualBasicTest/RemoveUnnecessaryImports/RemoveUnnecessaryImportsTests.vb

```diff
@@ -516,7 +515,7 @@ Namespace B
     Public Class CB
     End Class
 End Namespace|]",
-"Imports A
+"Imports A _
```

Comment:
Currently my bottleneck is the test-runner - tried a couple of other tools. It's really inconvenient to use command-line or wpf xunit runner. I'm wondering if either of those are what you're using for local testing.